lib/process-services-cloud/src/lib/form/form-cloud.module.ts

```diff
@@ -80,6 +81,9 @@ import { FilePropertiesTableCloudComponent } from './components/widgets/attach-f
         PeopleCloudWidgetComponent,
         GroupCloudWidgetComponent,
         PropertiesViewerWidgetComponent
+    ],
+    providers: [
+        { provide: ErrorStateMatcher, useClass: InvalidErrorStateMatcher }
```

Comment:
Are you sure we need to overwrite the provider in a module level? 
I think we need something like this in the html input element
```
<input 
[errorStateMatcher]="invalidErrorStateMatcher">
```

Reply:
I am sorry, I didn't see you were doing the same in core. I though you were overwriting it only in cloud which would result in APS not containing the fix. Ignore this comment please. Thanks